resource.go

```diff
@@ -98,7 +98,7 @@ func (r Resource) Groups() []string {
 		if v == "" {
 			g := k
 			groups = append(groups, g)
-		// Key-value
+			// Key-value
```

Comment:
Is this a gofmt change? I know it's not part of this PR, but since we're touching it, please move it inside the else branch or remove it.